Gemfile

```diff
@@ -59,7 +59,7 @@ group :development, :test do
   gem 'rubocop', '~> 0.42.0', require: false
 
   # Security scanners
-  gem 'brakeman'
+  gem 'brakeman', '3.4.1'
```

Comment:
Any reason to explicitly lock it to this version in the Gemfile? I think we should only do this if we specifically want to stick with that version going forward.